hydrus/metadata/drone_doc_gen.py

```diff
@@ -14,15 +14,15 @@ def drone_doc(API, BASE_URL):
                        BASE_URL)
 
     # Status Class
-    status = HydraClass("Status", "Status", "Class for drone status objects")
+    status = HydraClass("http://hydrus.com/Status", "Status", "Class for drone status objects")
```

Comment:
Why do we need the http://hydrus.com/ prefix? Status is defined in the API doc. Why have it dereference to a non existent URI? 
vocab:Status should be fine I think.